.gitignore

```diff
@@ -4,6 +4,8 @@ __pycache__/
 *$py.class
 
 # C extensions
+numcodecs/**/*.c
+numcodecs/**/*.h
```

Comment:
Won't C files be generated by Cython? Shouldn't the resulting files be ignored by version control?